docs/connection/db_connection/clickhouse/types.rst

```diff
@@ -3,6 +3,16 @@
 Clickhouse <-> Spark type mapping
 =================================
 
+.. note::
+
+    The results below are valid for Spark 3.5.3, and may differ on other Spark versions.
+
+.. note::
+
+    It is recommended to use `spark-dialect-extension <https://github.com/MobileTeleSystems/spark-dialect-extension>`_ package,
```

Comment:
may be provide code block in pyspark with example how to create spark session with added .jar file and registration of dialect via java classes?
 (or at least add example with registar dialect in `spark-dialect-extension` existing documentation?)

Reply:
I can update all Clickhouse documentation to include this package

Reply:
imho it is not redundant, but not in onetl docs reference, neither in spark-dialect-extension documentation persists usage example with pyspark regestring dialect 
(I thought that adding code block to onetl is more relevant, as users do not have to look in scala repo and sort out mixed examples in scala and python)